cli.sh

```diff
@@ -1897,6 +1983,126 @@ config_update_last_run() {
     fi
 }
 
+#####################################################
+# Manages the app script folders.
+#
+# Returns:
+#   1 on failure, 0 otherwise
+#####################################################
+manage_apps() {
+    if [ "$1" = "help" ]; then
+        print_help_header
+        echo "
+git hooks apps install download
```

Comment:
I'd change this to `git hooks tools register <type> <script>` where `<type>` would be `download` or similar here, and you could point to any file with `<script>` that gets copied into the global folder we want it in.

Reply:
Yeah, I suppose that makes sense. But is this something you actually need at the moment? I'd start with a simple single file for now, and add extra bits only if needed. Codewise it's not much of a difference, but too many options can get confusing to people.